crates/red_knot_python_semantic/resources/mdtest/call/getattr_static.md

```diff
@@ -59,7 +59,7 @@ import sys
 reveal_type(inspect.getattr_static(sys, "platform"))  # revealed: LiteralString
 reveal_type(inspect.getattr_static(inspect, "getattr_static"))  # revealed: Literal[getattr_static]
 
-reveal_type(inspect.getattr_static(1, "real"))  # revealed: Literal[real]
+reveal_type(inspect.getattr_static(1, "real"))  # revealed: property
```

Comment:
In an example like
```py
import inspect

class C:
    @property
    def p(self):
        return 1

reveal_type(inspect.getattr_static(C(), "p"))
```
we would previously get `Literal[p]`, and now we get `property`, which is exactly what the runtime version of `getattr_static` does as well (return the property instance). So this change looks fine to me, since `real` is a `@property` on class `int`. At runtime, `getattr_static` returns `<attribute 'real' of 'int' objects>` of type `getset_descriptor`; but I don't think we need to model that here. Revealing `property` seems closer to what we had before, in any case.